<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewUserDialog</class>
 <widget class="QDialog" name="CreateNewUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="name_edig">
     <property name="placeholderText">
      <string>Name of new user</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="email_edit">
     <property name="placeholderText">
      <string>Email address for new user</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="may_approve_expenses_checkbox">
     <property name="text">
      <string>May approve expenses</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="may_manage_users_checkbox">
     <property name="text">
      <string>May manage users</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="keep_dialog_open_checkbox">
       <property name="text">
        <string>Keep this dialog open</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="create_button">
       <property name="text">
        <string>&amp;Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>C&amp;ancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
